Library/Formula/gneural-network.rb

```diff
@@ -0,0 +1,24 @@
+class GneuralNetwork < Formula
+  desc "GNU Gneural Network implements a programmable neural network."
+  homepage "https://www.gnu.org/software/gneuralnetwork/"
+  url "http://cvs.savannah.gnu.org/viewvc/*checkout*/gneuralnetwork/gneural_network-0.5.0.tar.gz?revision=1.1&root=gneuralnetwork"
+  version "0.5.0"
+  sha256 "7bfff0cc3e4a1fd7ffa18c5c2476bd6d93ffb0a14b50c7feda9312bea943cf9f"
+  
+  depends_on "automake" => :build
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "aclocal"
```

Comment:
This is an odd series of steps; usually if autotools are required it's because `configure` doesn't exist.  Are you sure this line and the next are necessary?